testing/kuttl/source/cluster-start/01-psql-connect.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: psql-connect
+spec:
+  template:
+    spec:
+      restartPolicy: "OnFailure"
+      containers:
+        - name: psql
+          image: ${KUTTL_PSQL_IMAGE}
```

Comment:
I wonder if we could point to the latest upgrade image or even build our testing image? All we need in the container are the clients, psql, pgbackrest, krb5, etc. :thinking:

Reply:
But wouldn't we want the psql client to be the same version as the PG pod?

Reply:
Not necessarily, I think Postgres allows you to use a newer client version with an older server. 

https://www.postgresql.org/docs/14/app-psql.html
> psql works best with servers of the same or an older major version

Reply:
Just tested and can confirm that psql 14 can work with PG 13 servers. But after this morning's discussion, the decision was made not to create a testing image; the upgrade image might be worth considering, but for now I think it might be useful to leave this as a env var.